src/webui/www/private/views/preferences.html

```diff
@@ -1018,6 +1018,26 @@
                     <input type="checkbox" id="allowMultipleConnectionsFromTheSameIPAddress" />
                 </td>
             </tr>
+            <tr>
+                <td>
+                    <label for="validateHTTPSTrackers">QBT_TR(Validate HTTPS Trackers:)QBT_TR[CONTEXT=OptionsDialog]</label>
+                </td>
+                <td>
+                    <input type="checkbox" id="validateHTTPSTrackers" />
+                </td>
+            </tr>
+            <tr>
+                <td>
+                    <label for="sslVersion">QBT_TR(TLS version for HTTPS trackers (requires restart):)QBT_TR[CONTEXT=OptionsDialog]</label>
+                </td>
+                <td>
+                    <select id="sslVersion" style="width: 15em;">
+                        <option value="0">QBT_TR(TLSv1.1)QBT_TR[CONTEXT=OptionsDialog]</option>
+                        <option value="1">QBT_TR(TLSv1.2)QBT_TR[CONTEXT=OptionsDialog]</option>
+                        <option value="2">QBT_TR(TLSv1.3)QBT_TR[CONTEXT=OptionsDialog]</option>
```

Comment:
Add a note here explaining that TLSv1.3 will silently default to v1.2 in case v1.3 is not supported.